metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-rest-env.xml

```diff
@@ -35,32 +35,29 @@
     </property>
     <property>
         <name>metron_spring_profiles_active</name>
-        <description>Active Spring profiles</description>
+        <description>Active Spring profiles. 'ldap' is used to enable authentication via LDAP.</description>
         <display-name>Active Spring profiles</display-name>
-        <value/>
-        <value-attributes>
-            <empty-value-valid>true</empty-value-valid>
-        </value-attributes>
+        <value>jdbc</value>
```

Comment:
Are we using a "jdbc" profile anywhere?  From what I can tell jdbc authentication is used by default but we don't actually use a profile for that.

Reply:
good catch. I thought I'd rolled that back.  I'll udpate